src/Hamelin.Runtimes.GitHubActions/Hamelin.Runtimes.GitHubActions.csproj

```
@@ -14,7 +14,7 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/hamelin-org/Hamelin.Runtimes.GitHubActions.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <Version>1.0.1</Version>
+    <Version>1.1.0</Version>
```

```suggestion
    <Version>2.0.0</Version>
```
I think the changes to the public `GitHubActionsCommands` class are backwards-incompatible? The parameterless constructor is gone as are some static methods.

---

Unless, I missed something, the only static method I can see being removed was private?

And the constructor change is technically breaking, in the same way that that adding `WithGroup` to the `IGitHubActionsCommands` interface is, but it was never intended to be instantiated manually.

Should we just make the whole class `internal` instead?

---

Decided to make it `internal` and do the major version bump.